index.html

```diff
@@ -178,5 +178,58 @@ <h4 class="skill-title">Skills</h4>
       </div>
     </div>
   </section>
+  <section class="section-contact-form">
+    <div class="contact-form-container grid gen-margin">
+      <h3 class="contact-form-primary">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
+      <form action="" class="contact-form">
```

Comment:
- [ ] Your form is not submitting.
- Kindly use the action and method attribute to integrate with [formspree service](https://formspree.io/html). 
* `action` value should be similar to `https://formspree.io/f/{form_id}`
* `method` value equals `POST`